lib/convict.js

```diff
@@ -401,7 +407,7 @@ var convict = function convict(def) {
       var self = this;
       if (!Array.isArray(paths)) paths = [paths];
       paths.forEach(function(path) {
-        overlay(cjson.load(path), self._instance, self._schema);
+        overlay(loadJSON(path), self._instance, self._schema);
```

Comment:
I try to replace the line with
```javascript
        fs.readFile(path, 'utf-8', function(err, data) {
          if (err) throw err;
          overlay(json5.parse(data.toString()), self._instance, self._schema);
        });
```
The test fails, seems `loadFile` has to be a sync call.